spec/features/user_signs_out_spec.rb

```diff
@@ -17,10 +17,10 @@
     fill_in 'Email', with: user.email
     fill_in 'Password', with: user.password
     click_button 'Log in'
-    expect(page).to have_content('Signed in successfully')
+    expect(page).to have_content('Welcome')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.